stack.yaml

```diff
@@ -0,0 +1,61 @@
+resolver: lts-5.13
```

Comment:
This is one reason I've been loathe to add `stack.yaml` files to Diagrams repos.  To use stack with the way we develop Diagrams requires picking a directory layout.  Do we go with `../diagrams-foo`, or do we remove the `diagrams-` prefix?

Without automation, I think git submodules or commit IDs in the `stack.yaml` will be much too annoying.  Too time consuming for the amount of time we can spend on Diagrams.  (I'd like to see this automation, to get reproducible builds, but that's a separate issue.)